api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -329,7 +329,7 @@ private void maybeAlertOnCostThresholds(
 
     if (remainingBalance < 0 && previousRemainingBalance > 0) {
       try {
-        mailService.alertUserFreeTierExpiration(user);
```

Comment:
TODO (too big for this PR): rename all "free tier" instances in FreeTierBillingService, plus the service itself